src/components/ArticleComponents/AuthorList/index.tsx

```diff
@@ -1,23 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { FormattedMessage } from 'react-intl';
 import Author from './Author';
 import styles from './index.module.scss';
+import githubFallbackImg from '../../../images/placeholder-img.png';
+
+const GITHUB_IMG_SIZE = '60';
 
 interface Props {
   authors: string[];
 }
 
+const authorImgCache = {};
+
+function checkAuthorImg(username: string): Promise<string> {
+  if (authorImgCache[username]) {
+    return authorImgCache[username];
+  }
+  return new Promise(resolve => {
```

Comment:
I think this is a little bit too much. You can use CSS for fallback images.

Reply:
So instead of using an `<img>` tag, you can use a `<span>` one and use CSS background. https://stackoverflow.com/a/20684707

Reply:
Yeah we can do that